### PR TITLE
proper versioning with `setuptools_scm`

### PR DESCRIPTION
**Pull request by jGaboardi**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

xref: https://github.com/conda-forge/libpysal-feedstock/pull/20#issuecomment-1742605869

### CODE REVIEW

**Comment by jGaboardi** [comment]
@conda-forge-admin, please rerender

**Comment by jGaboardi** [comment]
Also, @knaaptime you can add me to be automatically tagged for PR reviews in this repo if you like.

**Comment by knaaptime** [comment]
thanks for this. I cut a new release today, so i think another PR will get opened by the bot here before long. (and thanks! how do i add you as autoreview?)

**Comment by jGaboardi** [comment]
Seems I don't have admin privileges here, Also, it's strange that I am listed in the maintainers section of `meta.yml`, but not in the `README.md`.

**Comment by jGaboardi** [comment]
> Also, it's strange that I am listed in the maintainers section of meta.yml, but not in the README.md.

Seems that the previous version was never re-rendered somehow. I am listed in the `README.md` as a maintainer now.

**Comment by jGaboardi** [comment]
Let's get this merged before #24 to see proper versioning.

**Comment by knaaptime** [comment]
i was thinking we just edit the impending PR before it automerges and we can inlude these fixes without incrementing the build

**Comment by knaaptime** [comment]
but im game to merge this too. assume everything passed?

**Comment by jGaboardi** [comment]
I am wondering if that PR is forthcoming or not. Looks like the bot never picked it up [over in `mapclassify`](https://github.com/conda-forge/mapclassify-feedstock/issues/21).

**Comment by jGaboardi** [comment]
> but im game to merge this too. assume everything passed?

Yep.

**Comment by knaaptime** [comment]
weird. I'm not sure what the deal is with mapclassify, but usually it just takes a few hours after its on pypi. I'd expect this one sometime late-ish tonight

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,21 +24,15 @@ requirements:
...
-    - dash
-    - dash-bootstrap-components
-    - palettable
-    - pip
     - tqdm
     - quilt3 >=3.1.11
     - xlrd
+    - appdirs
     - mapclassify
     - region >=0.2.0
```
Comment (jGaboardi) [inline recipe/meta.yaml]: Is `region` still a dependency?

Comment (knaaptime) [reply recipe/meta.yaml]: no, missed that one. Feel free to rm and commit directly here

**Comment by jGaboardi** [comment]
@conda-forge-admin, please rerender

**Comment by knaaptime** [comment]
also i think the scm_versioning need to go in host requirements, not deps

**Comment by jGaboardi** [comment]
> also i think the scm_versioning need to go in host requirements, not deps

[You are correct](https://github.com/conda-forge/libpysal-feedstock/pull/20/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR17). There are now multiple feedstock PRs I have screwed up today...

**Comment by knaaptime** [comment]
:P it took me 4 RCs to get the docs building

**Pull request merged**